### ISSUE TITLE
Quest: Burning Shadows

### ISSUE THREAD

**Issue by underlord-oberon** [plantain-gnocchis/wow-voiceover-tbc-wotlk#76]
Add-on version: AI_VoiceOver_v1.4.5
Data version: AI_VoiceOverData_VanillaExtra_v1.4

Durotar / Skull Rock / Item: Eye of Burning Shadow / Quest: Burning Shadows
Link: https://www.wowhead.com/wotlk/quest=832/burning-shadows

Description: the voice-over suddenly silences during playback. This text is also a telepathic communication from other member of the Burning Blade. Perhaps an orc or demon voice would be more suitable instead of the narrator voice.

Progress: no playback.

Completion: no playback.

**Comment by carevass** [comment]
This should be fixed with new audio in AI_VoiceOverData_VanillaExtra_v1.5.
